xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/editpanels.vm

```diff
@@ -36,7 +36,7 @@
 #end
 #if ($!editorPanels != '')
   #set ($editorPanels = $editorPanels.split(','))
-  <div id="editPanels" class="panels editor">
+  <div id="editPanels" class="panels editor xform">
```

Comment:
Like I told you earlier, this is not the ideal solution.

You should add the `xform` class to the form so that all other forms (if any) get covered by this.

I think the right file would be: [edit.vm](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm)

Reply:
Oh yes.. I see! Thanks. I will do this again!